CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ### Added
 
 - Add the new `go.opentelemetry.io/contrib/instrgen` package to provide auto-generated source code instrumentation. (#3068, #3108)
+- Remove the `SemVersion` function (#3681)
```

Comment:
1. This should be in "Removed" section (not added)
2. We should add in "Changed" something like: `semver: ` prefix is removed from version of the instrumentation scope.

Also

```suggestion
- Remove the `SemVersion` functions in instrumentation packages. (#3681)
```

Reply:
Just done, thanks your comment @pellared

Reply:
Point 2 has not been addressed.

Reply:
Please checkout again